make/platformio.sconscript

```diff
@@ -4260,8 +4260,16 @@ source_files = []
 for src in env.LookupSources(variant_dir, src_dir, True, src_filter):
     source_files.append(env.Object(src))
 
+# get settings file path
+settings_ini_path = ARGUMENTS.get('SETTINGS_PATH', "").decode('base64')
```

Comment:
Rename `settings_ini_path` and `settings_path` to `settings_ini`, as that is how the make variable is called in the make build system, just to make the naming consistent throughout the code base. `settings_path` may be a better name let's not change it to that now.